admin/src/app/Components/registration/registration.component.html

```diff
@@ -0,0 +1,39 @@
+<h1>Add new admin</h1>
+<form mat-dialog-content [formGroup]="registrationForm">
+    <p>Enter <b>new admin</b> email</p>
+    <mat-form-field apperance="fill" style="width: 95%">
+        <mat-label>email</mat-label>
+        <input 
+            matInput
+            formControlName="email"
+            type="email"
+            placeholder="new_admin@gmail.com"
+        >
+        <mat-error *ngIf="email?.invalid">
+            Please enter a valid email address
+        </mat-error>
+    </mat-form-field>
+    <p>Enter <b>new admin</b> password</p>
+    <mat-form-field appearance="fill" style="width: 95%;">
+        <mat-label>password</mat-label>
+        <input 
+            matInput
+            formControlName="password"
+            type="text"
+        >
+        <mat-error *ngIf="password?.invalid">
+            Password is required
+        </mat-error>
+    </mat-form-field>
+    <button mat-button (click)="closeRegistrationDialog()">Back</button>
+    <button
+        [disabled]="registrationForm.invalid"
+        mat-button 
+        (click)="register(email?.value, password?.value)"
+    >
+        Register
+    </button>
+    <mat-error *ngIf="unsuccessWarning">
```

Comment:
Is there any success warning?

Reply:
Was redone with server answer displaying